### PR TITLE
[OYS-84]: Open Y Google Search search results page must be always cre…

### PR DESCRIPTION
**Pull request by AleksandrPrykhodkoDev**
### Steps for review

- [ ] Login as admin
- [ ] Go to /admin/openy/settings/google-search and set Google Engine ID
- [ ] Go back to the homepage and run search for something with the header search form
- [ ] Verify the user is redirected to the landing with search results
- [ ] Edit the landing with the results paragraph.
- [ ] Change the path alias
- [ ] Go back to the homepage and run a search again.
- [ ] Verify the user is redirected to the new alias and search results are displayed.

### CODE REVIEW

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

File: modules/custom/openy_search/openy_google_search/openy_google_search.install
Comment (dchubar-fj) [inline modules/custom/openy_search/openy_google_search/openy_google_search.install]: Please don't use hardcoded language see your PR https://github.com/ymcatwincities/openy/pull/1921

**Review by dchubar-fj — CHANGES_REQUESTED** [review]

File: modules/custom/openy_search/openy_google_search/openy_google_search.install
Comment (AleksandrPrykhodkoDev) [reply modules/custom/openy_search/openy_google_search/openy_google_search.install]: Fixed

**Review by dchubar-fj — DISMISSED** [review]

**Comment by podarok** [comment]
@AleksandrPrykhodkoDev please fix conflict and verify with @AndreyMaximov why PR DO NOT MERGE still exists.
Thank you

**Review by podarok — DISMISSED** [review]
I see this is a fixed. @AndreyMaximov your approval needed to unblock PR's labels in GitHub

**Comment by AndreyMaximov** [comment]
ok to test

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2074/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2074/

**Comment by AndreyMaximov** [comment]
@podarok the PR still needs work, the label is correct.

We discussed the task internally - not all acceptance criteria are met and the steps for review don't reflect them.

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2074/

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2077/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2077/

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2077/

File: modules/custom/openy_search/src/Config/OpenySearchOverrides.php
Comment (dchubar-fj) [inline modules/custom/openy_search/src/Config/OpenySearchOverrides.php:53]: We should not completely remove the overriding of `search_page_alias` config. We should replace hardcoded `search` alias with correct alias from the search results landing node.

**Review by dchubar-fj — CHANGES_REQUESTED** [review]

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2079/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2079/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2079/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2079/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2079/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2079/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2079/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2079/

**Review by afi13 — APPROVED** [review]

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2081/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2081/

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2081/

**Comment by AndreyMaximov** [comment]
@podarok We decided to incorporate these changes in Search API related PR.

**Pull request closed without merging**
